### PR TITLE
add stylable-react-component

### PR DESCRIPTION
**Pull request by wtfil**
Added `stylable-react-componet` to be able to use component decorators and `cssStates` property on components

### CODE REVIEW

File: test/utils/has-state.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {expect} from 'test-drive-react';
+export default function hasState(elem: Element | null, styles: any, name: string): void | never {
```
Comment (Ta4i) [inline test/utils/has-state.ts]: State is React keyword, so maybe it's better to rename this function to hasCSSState/hasCustomState/<your option>?

Comment (wtfil) [inline test/utils/has-state.ts]: good point

**Comment by AviVahl** [comment]
I'm merging this. Great work.
The `hasCssState` function will be taken into `stylable-integration/test-utils`.

**Comment by tyv** [comment]
🎉

**Pull request merged**